advance_check_void/wizard/print_prenumbers_checks.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from odoo import _, models
+from odoo.exceptions import ValidationError
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class PrintPreNumberedChecks(models.TransientModel):
+    _inherit = "print.prenumbered.checks"
```

Comment:
This does not seem related to voiding checks, as advertised in the module title.
Maybe it is a different feature and a different module?

Reply:
I agree, this can be removed from this module.

Reply:
Maybe make it a second module, to not lose the feature.